frontend/test/pytest/test_device_api.py

```diff
@@ -60,8 +60,10 @@ def execute(self, circuits, execution_config):
         """Execution."""
         return circuits, execution_config
 
-    def preprocess(self, execution_config: ExecutionConfig = DefaultExecutionConfig):
+    def preprocess(self, execution_config=None):
```

Comment:
Joey has found the underlying issue in the meantime (the mutation), but I just want to note that this is the original signature of the base class.